idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1799 @@
+                'mole_frac_comp':
+                {'H2O': blk.vap_in_mole_frac[0, 'H2O'].value,
+                 'CO2': blk.vap_in_mole_frac[0, 'CO2'].value}}
+
+        liquid_phase_state_args = {
+            'flow_mol': blk.liq_in_flow[0].value,
+            'temperature': blk.liq_in_temperature[0].value,
+            'pressure': blk.bot_pressure[0].value,
+            'mole_frac_comp':
+            {'H2O': blk.liq_in_mole_frac[0, 'H2O'].value,
+             'CO2': blk.liq_in_mole_frac[0, 'CO2'].value,
+             'MEA': blk.liq_in_mole_frac[0, 'MEA'].value}}
+
+        print('==============================================================')
+        print("STEP 1: Property Package initialization")
+        print('==============================================================')
+
+        # Initialize vapor_phase block
+        blk.vapor_phase.properties.initialize(
+            state_args=vapor_phase_state_args,
+            state_vars_fixed=False,
+            outlvl=outlvl,
+            optarg=optarg,
+            solver=solver,
+            hold_state=True)
+        # Initialize liquid_phase properties block
+        blk.liquid_phase.properties.initialize(
+            state_args=liquid_phase_state_args,
+            state_vars_fixed=False,
+            outlvl=outlvl,
+            optarg=optarg,
+            solver=solver,
+            hold_state=True)
+
+        print('==============================================================')
+        print("STEP 2: Steady-State ISOTHERMAL MASS BALANCE")
+        print('        -->no mass transfer ')
+        print('        -->no heat transfer')
+
+        # unfix flow variable terms
+        # vapor side
+        blk.vapor_phase.properties[:, :].flow_mol.unfix()
+        blk.vapor_phase.properties[:, :].mole_frac_comp[:].unfix()
+        blk.vapor_phase._flow_terms[:, :, :, :].unfix()
+        blk.vapor_phase.material_flow_dx[:, :, :, :].unfix()
+        # liquid-side
+        blk.liquid_phase.properties[:, :].flow_mol.unfix()
+        blk.liquid_phase.properties[:, :].mole_frac_comp[:].unfix()
+        blk.liquid_phase._flow_terms[:, :, :, :].unfix()
+        blk.liquid_phase.material_flow_dx[:, :, :, :].unfix()
+
+        # activate mass balance related equations
+        for c in blk.component_objects(Constraint, descend_into=True):
+            if c.local_name in [
+                "eq_sumy",
+                "eq_sumx",
+                "liq_BC_mole_frac",
+                "vap_BC_mole_frac",
+                "vap_BC_flow_mol",
+                "liq_BC_flow_mol",
+                "material_balances",
+                "material_flow_linking_constraints",
+                    "material_flow_dx_disc_eq"]:
+                c.activate()
+
+        # solve for a small lenght if stripper
+        if (blk.config.process_type == 'Stripper' or
+                blk.config.process_type == 'Regenerator'):
+            specified_length = value(blk.length_col)
+            blk.length_col.fix(0.6)
```

Comment:
After some analyses, I discovered that it is easier to initialize the column for stripper condition at a small scale before scaling it up.  The degrees of freedom is zero.

Reply:
OK, I see what you are doing now. Again, I would suggest using the homotopy tool later on when you come to scaling up.

For the future however, this might suggest a scaling issue which would be better solved using the scaling tools.